src/test/groovy/graphql/schema/diffing/ana/EditOperationAnalyzerAppliedDirectivesTest.groovy

```diff
@@ -547,7 +547,11 @@ class EditOperationAnalyzerAppliedDirectivesTest extends Specification {
         def changes = calcDiff(oldSdl, newSdl)
         then:
         changes.enumDifferences["E"] instanceof EnumModification
-        def appliedDirective = (changes.enumDifferences["E"] as EnumModification).getDetails(AppliedDirectiveDeletion)
+        def diff = changes.enumDifferences["E"] as EnumModification
+
+        diff.getDetails().size() == 1
```

Comment:
I haven't had time to change the rest, but the tests should really ensure all details are checked.

In this test we previously had an undetected change where the deleted directive also introduced a enum value removed change detail.